scripts/arangodb-docker-container-setup/docker-container-scripts/test-data/testData.json

```diff
@@ -438,11 +449,26 @@
             ],
             "note": "this section is the best part of an illustrated book about birds"
         },
+        {
+            "_to": "bibliographic_references/1",
+            "_from": "bibliographic_references/1",
+            "type": "self",
+            "_key": "9",
+            "members": [
+                {
+                    "role": "self",
+                    "context": {
+                        "type": "general"
+                    }
+                }
+            ],
+            "note": "found this in the archives"
+        },
         {
             "_to": "terms/2",
             "_from": "terms/2",
             "type": "self",
-            "_key": "9",
+            "_key": "10",
```

Comment:
The diff makes a lot more sense than the snippets jest was showing me. This looks good.